modules/tinymce/src/plugins/table/test/ts/browser/command/MergeCellCommandTest.ts

```diff
@@ -35,68 +31,153 @@ UnitTest.asynctest('browser.tinymce.plugins.table.command.MergeCellCommandTest',
   const clearEvents = () => modifiedEvents = [];
   const defaultEvent = { type: 'tablemodified', structure: true, style: false } as TableModifiedEvent;
 
-  const testCommand = (editor: Editor, command: string, tests: MergeCellTest[]) => {
-    Tools.each(tests, (test) => {
-      clearEvents();
-      editor.getBody().innerHTML = test.before;
-      editor.selection.select(editor.dom.select('td[data-mce-selected]')[0], true);
-      editor.selection.collapse(true);
-      editor.execCommand(command);
-      LegacyUnit.equal(cleanTableHtml(editor.getContent()), test.after, test.message);
-      LegacyUnit.equal(modifiedEvents, test.expectedEvents);
-    });
+  const testMerge = (editor: Editor, test: MergeCellTest) => {
+    clearEvents();
+    editor.getBody().innerHTML = test.before;
+    editor.selection.select(editor.dom.select('td[data-mce-selected]')[0], true);
+    editor.selection.collapse(true);
+    editor.execCommand('mceTableMergeCells');
+    LegacyUnit.equal(cleanTableHtml(editor.getContent()), test.after);
+    assert.deepEqual(modifiedEvents, test.expectedEvents);
   };
 
   const cleanTableHtml = (html: string) => {
     return html.replace(/<p>(&nbsp;|<br[^>]+>)<\/p>$/, '');
   };
 
-  suite.test('TestCase-TBA: Table: mceTableMergeCells', (editor) => {
-    testCommand(editor, 'mceTableMergeCells', [
-      {
-        message: 'Should merge all cells into one',
-        before: (
-          '<table>' +
+  const getWidthNumber = (widthStyle: string): number => Number(widthStyle.replace('%', ''));
+
+  const hook = TinyHooks.bddSetupLight<Editor>({
```

Comment:
Can you please put the hook at the top of the file (ie before all the helper functions), as that's where we've put it in every other test so far.